src/parser.ts

```diff
@@ -32,6 +30,7 @@ export interface PropItem {
   description: string;
   defaultValue: any;
   parent?: ParentType;
+  parents?: ParentType[];
```

Comment:
I am not sure if the `parents` is the best name.  Don't you think it can be confusing with `parent` property?

Reply:
I had thoughts that `parents` is confusing a little bit. How do you feel about `declarations` or `origins`?

Reply:
I think `declarations` sounds pretty good.